css/style.css

```diff
@@ -0,0 +1,88 @@
+@import url('https://fonts.googleapis.com/css2?family=Chicle&family=Poppins:wght@300;400;500;600;700;800;900&family=Roboto:ital,wght@0,400;0,500;0,700;0,900;1,900&display=swap');
+
+body {
+  padding: 0;
+  margin: 0;
+}
+
+.MainContainer {
+  width: 100%;
+  height: 100vh;
+}
+
+header {
```

Comment:
- [ ] [OPTIONAL] Please stick to the design as much as possible, including the margin/padding, width, and size

|Your design|Figm design|
|---|---|
|![Screenshot_20220525_200623](https://user-images.githubusercontent.com/25227022/170349497-81b13807-821b-4711-86f4-b9b9afd6d777.png)|![Screenshot_20220525_201307](https://user-images.githubusercontent.com/25227022/170349684-9ec3ab28-166f-452f-ace3-b8a4e5ab8781.png)|